InvenTree/InvenTree/settings.py

```diff
@@ -608,7 +608,7 @@
 
 # Magic login django-sesame
 SESAME_MAX_AGE = 300
-LOGIN_REDIRECT_URL = "/platform/logged-in/"
+LOGIN_REDIRECT_URL = "/logged-in/"
```

Comment:
@SchrodingersGat pretty sure this endpoint does not exist - `/plattform/logged-in/` is a jsx endpoint on P UI, not a Django endpoint